### PR TITLE
Popup page

### PR DESCRIPTION
**Pull request by Bushido-brown**
I used script.js to create a popup window for my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,7 +81,7 @@ <h1 class="font">
         </div>
       </section>
       <div class="works" id="portfolio">
-        <div class="showcase-1 showcase">
+        <!-- <div class="showcase-1 showcase">
           <div class="showcase-partition img-content">
             <img
               src="img/Snapshoot-Portfolio.png"
```
Comment (mahbubul14) [inline index.html]: - [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly remove the inline comments from your code(**line 84 - 191**). :+1:

File: index.html [hunk 2]
```diff
@@ -80,7 +81,7 @@ <h1 class="font">
         </div>
       </section>
       <div class="works" id="portfolio">
```
Comment (mahbubul14) [inline index.html]: [ ] Currently, your design is not matched with the Figma design. Kindly reduce the gap between the main page and the pop-up and try to match with the Figma design. :+1:

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y2 | f2


- [ ] Kindly do the adjustment for mobile also. :+1:

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y3 | f3

- [ ] Please try to match the font color according to the Figma design. :+1:

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y1 | f1

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Bushido-brown ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] In the `README` file, Kindly update the "Getting Started" section. This should be an example of how you may give instructions on setting up your project locally.

- [ ] Kindly check the inline comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter also should be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**